doc/release-notes.md

```diff
@@ -168,19 +168,32 @@ Low-level Changes section below.
   progress of the wallet's scanning historical blocks for transactions
   affecting its balances. (#15730)
 
+- `gettransaction` now accepts a third (boolean) argument `verbose`. If
+  set to `true`, a new `decoded` field will be added to the response containing
+  the decoded transaction. This field is equivalent to RPC `decoderawtransaction`,
+  or RPC `getrawtransaction` when `verbose` is passed.
+
 - `createwallet` accepts a new `passphrase` parameter.  If set, this
   will create the new wallet encrypted with the given passphrase.  If
   unset (the default) or set to an empty string, no encryption will be
   used. (#16394)
 
+- `getchaintxstats` RPC now returns the additional key of
+  `window_final_block_height`.
+
 - `getmempoolentry` now provides a `weight` field containing the
   transaction weight as defined in BIP141. (#16647)
 
+- The `getnetworkinfo` and `getpeerinfo` commands now contain a new field with decoded network service flags.
+
 - `getdescriptorinfo` now returns an additional `checksum` field
   containing the checksum for the unmodified descriptor provided by the
   user (that is, before the descriptor is normalized for the
   `descriptor` field). (#15986)
 
+- `joinpsbts` will shuffle the order of the inputs and outputs of the resulting joined psbt.
+  Previously inputs and outputs were added in the order that the PSBTs were provided which makes correlating inputs to outputs extremely easy.
```

Comment:
Suggestion: "Previously, inputs and outputs were added in the order PSBTs were provided. This made it easy to correlate inputs to outputs, representing a privacy leak."

Reply:
I'd prefer to keep this move-only. It can be changed in the wiki by anyone, once they are moved: https://github.com/bitcoin-core/bitcoin-devwiki/wiki/0.19.0-Release-Notes-Draft